dusty/cli/upgrade.py

```diff
@@ -0,0 +1,25 @@
+"""Upgrade Dusty's binaries
+
+If Dusty is being run with a binary, this command will replace the
+binary with the specified version from GitHub.  If no version is
+specified, the latest version of the binary will be used.  The
+daemon will then exec into the new version of the binary.
```

Comment:
Use more user-facing language here, talk about upgrading the installed version of Dusty, not "the binary"